.github/workflows/ephemeral-env.yml

```diff
@@ -79,6 +79,80 @@ jobs:
           })
           core.setFailed(errMsg)
 
+  docker_ephemeral_env:
+    needs: config
+    if: needs.config.outputs.has-secrets
+    name: Push ephemeral env Docker image to ECR
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: "Download artifact"
+        uses: actions/github-script@v3.1.0
+        with:
+          script: |
+            const artifacts = await github.actions.listWorkflowRunArtifacts({
+                owner: context.repo.owner,
+                repo: context.repo.repo,
+                run_id: ${{ github.event.workflow_run.id }},
```

Comment:
hey @dpgaspar this change broke the ephemeral envs because the context is different. Before the trigger was a workflow_run, and now being comment created this property is not available in this context.